File: java/sui-user-transaction/src/main/java/ru/sui/usertransactionconfigurier/autoconfigure/AutoConfiguration.java
package ru.sui.usertransactionconfigurier.autoconfigure;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.core.Ordered;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@ComponentScan("ru.sui.usertransactionconfigurier")
@EnableTransactionManagement(order = Ordered.LOWEST_PRECEDENCE - 1)
public class AutoConfiguration {

}

File: java/sui-security/base/src/main/java/ru/sui/suisecurity/base/config/AutoConfiguration.kt
package ru.sui.suisecurity.base.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.scheduling.annotation.EnableScheduling
import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity


@Configuration
@EnableWebSecurity
@EnableGlobalMethodSecurity(securedEnabled = true, jsr250Enabled = true, prePostEnabled = true, proxyTargetClass = true)
@ComponentScan("ru.sui.suisecurity.base")
@EnableScheduling
class AutoConfiguration
